handlers/graph_unused.go

```diff
@@ -0,0 +1,144 @@
+package handlers
+
+import (
+	"fmt"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/graph/tree"
+	"github.com/kiali/kiali/log"
+	"k8s.io/api/apps/v1beta1"
```

Comment:
trivial comment - please order the imports the way the rest of the code base is. 

```
import (
  // std imports
  // blank line
  // third party imports
  // blank line
  // our kiali imports
```